common:
  global_frame_id: /map
  robot_base_frame_id: /base_link